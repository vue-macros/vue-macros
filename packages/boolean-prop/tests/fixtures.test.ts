import { resolve } from 'node:path'
import {
  rollupBuild,
  RollupVue,
  testFixtures,
  UnpluginOxc,
} from '@vue-macros/test-utils'
import { describe } from 'vitest'
import BooleanProp from '../src/vite'

describe('fixtures', async () => {
  await testFixtures(
    'tests/fixtures/*.{vue,[jt]s?(x)}',
    (args, id) =>
      rollupBuild(id, [BooleanProp(), RollupVue(), UnpluginOxc.rollup()]),
    {
      cwd: resolve(__dirname, '..'),
      promise: true,
    },
  )
})
