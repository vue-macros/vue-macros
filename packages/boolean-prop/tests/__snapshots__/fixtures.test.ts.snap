// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`fixtures > tests/fixtures/basic.vue 1`] = `
"// basic.js
import { createElementBlock, openBlock } from 'vue';
import _export_sfc from '[NULL]/plugin-vue/export-helper';

const _sfc_main = {};

const _hoisted_1 = {
  checked: true,
  enabled: false
};

function _sfc_render(_ctx, _cache) {
  return (openBlock(), createElementBlock("input", _hoisted_1))
}
var basic = /*#__PURE__*/_export_sfc(_sfc_main, [['render',_sfc_render],__FILE__]);

export { basic as default };
"
`;
